src/text.rs

```diff
@@ -355,13 +355,67 @@ pub fn draw_text_ex(text: &str, x: f32, y: f32, params: TextParams) {
                 dest_size: Some(vec2(dest.w, dest.h)),
                 source: Some(source),
                 rotation: angle_rad,
-                pivot: Option::Some(vec2(dest.x, dest.y)),
+                pivot: Some(vec2(dest.x, dest.y)),
                 ..Default::default()
             },
         );
     }
 }
 
+/// Draw multiline text with the given font_size, line_distance and color.
+/// If no line distance but a custom font is given, the fonts line gap will be used as line distance if it exists.
+pub fn draw_multiline_text(
+    text: &str,
+    x: f32,
+    y: f32,
+    font_size: f32,
+    line_distance: Option<f32>,
+    color: Color,
+) {
+    draw_multiline_text_ex(
+        text,
+        x,
+        y,
+        line_distance,
+        TextParams {
+            font_size: font_size as u16,
+            font_scale: 1.0,
+            color,
+            ..Default::default()
+        },
+    )
+}
+
+/// Draw multiline text with the given line distance and custom params such as font, font size and font scale.
+/// If no line distance but a custom font is given, the fonts line gap will be used as line distance if it exists.
+pub fn draw_multiline_text_ex(
+    text: &str,
+    x: f32,
+    mut y: f32,
+    line_distance: Option<f32>,
+    params: TextParams,
+) {
+    let font_line_distance = params
+        .font
+        .map(|font| {
+            font.font
+                .horizontal_line_metrics(1.0)
+                .map(|metrics| metrics.line_gap)
+                .unwrap_or(0.0)
+        })
+        .unwrap_or(0.0);
+    let line_distance = line_distance.unwrap_or(font_line_distance);
+    let max_height = text
```

Comment:
Doesn't it result in variable lines gap dependning on text content? I don't think that this is how fonts are supposed to work, but, honestly, I did not really researched the topic at all :( 

If you want I can look into this, I think there should be some good references on the topic!

Reply:
![image](https://github.com/not-fl3/macroquad/assets/85427285/50603598-46ed-481d-a686-d4902ca6fb3c)
This is what the fixed implementation looks like, it should be a constant line gap now.